src/main.rs

```diff
@@ -516,4 +524,26 @@ mod tests {
     drop(test_input_file);
     temp_dir.close().expect("Delete test directory");
   }
+
+  #[test]
+  fn handle_invalid_config() {
```

Comment:
Could you please modify this so it also works on Windows? It seems the stdio-override dependency does not compile on windows. Testing to see if any new files are created in the default output directory could be a potential alternative. Or maybe something else

Reply:
I see. It was my lack of knowledge about Rust. Let me update the test.